src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/FhirProfiles.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
```

Comment:
I don't think this should be in our mapper directory. Constants like this should be somewhere like the model directory